### PR TITLE
Replace deep imports with root imports

### PR DESCRIPTION
**Pull request by edufschmidt**
Change-type: patch

### CODE REVIEW

**Review by Ereski — APPROVED** [review]
LGTM :+1:

**Comment by Ereski** [comment]
the tests disagree with me tho @edufschmidt :p

```
lib/card.ts(188,6): error TS2554: Expected 2 arguments, but got 1.

lib/card.ts(188,6): error TS2801: This condition will always return true since this 'import("/tmp/build/66d12d10/versioned-source/node_modules/@types/bluebird/index")<any[]>' is always defined.

lib/card.ts(643,17): error TS2339: Property 'data' does not exist on type '"Object" | { <U>(onFulfill?: ((value: LinkConstraint[]) => Resolvable<U>) | undefined, onReject?: ((error: any) => Resolvable<U>) | undefined): Bluebird<...>; <TResult1 = LinkConstraint[], TResult2 = never>(onfulfilled?: ((value: LinkConstraint[]) => Resolvable<...>) | ... 1 more ... | undefined, onrejected?: ((reas...'.

  Property 'data' does not exist on type '"Object"'.
```

File: lib/index.ts [hunk 1]
```diff
@@ -1,30 +1,32 @@
 /* global FormData */
+import type { core, JSONSchema } from '@balena/jellyfish-types';
```
Comment (Ereski) [inline lib/index.ts:2]: What is being used from that `core` namespace? If it's just contracts that's fine but if it's something else you should replace those with the actual types.

**Comment by Ereski** [comment]
guess it's not LGTM

File: lib/index.ts (hunk 1 above)
Comment (Ereski) [reply lib/index.ts:2]: For `jellyfish-types` exceptionally it's fine to have deep imports. Makes it easier to see what is actually being imported. We'll be removing this module anyway.

Comment (edufschmidt) [reply lib/index.ts:2]: It's just `core.Contract`.

**Comment by edufschmidt** [comment]
> the tests disagree with me tho @edufschmidt :p

My bad @Ereski. Fixed an import and we should pass the tests now.

**Pull request merged**